routers/user/auth.go

```diff
@@ -1179,6 +1185,10 @@ func ResetPasswdPost(ctx *context.Context) {
 			ctx.ServerError("UpdateUser", err)
 			return
 		}
+
+		// Just in case the user is signed in to another account
+		handleSignOut(ctx)
```

Comment:
I would prefer that there would be check for code that was issued for what user and currently authorised user and if they do not much than return error (probably 404?)

Reply:
This could be another PR.